server/rooms.ts

```diff
@@ -1422,9 +1422,11 @@ export class GlobalRoomState {
 		const upperStaffRoom = Rooms.get('upperstaff');
 		let hasPrivateTerm = stack.includes('private');
 
-		const privateTerms = (Config.privatecrashterms ? [...Config.privatecrashterms] : []).map(item => new RegExp(item, "i"));
-		for (const term of privateTerms) {
-			if (term.test(stack) || term.test(err.message)) hasPrivateTerm = true;
+		for (const term of (Config.privatecrashterms || [])) {
+			if (typeof term === 'string' ? err.stack.includes(term) : term.test(err.stack)) {
```

Comment:
I don't know why GH Actions is having trouble displaying the correct error message, but I managed to get it by running tests on my own computer:

```
server/rooms.ts:1426:35 - error TS2532: Object is possibly 'undefined'.

1426  		if (typeof term === 'string' ? err.stack.includes(term) : term.test(err.stack)) {
      		                               ~~~~~~~~~
```